maestro-utils/src/main/kotlin/Insight.kt

```diff
@@ -0,0 +1,31 @@
+package maestro.utils
+
+object Insights {
+
+    private val insights = mutableSetOf<Insight>()
+    private var onInsightsUpdated: (List<Insight>) -> Unit = {}
+
+    fun report(insight: Insight) {
+        insights.add(insight)
```

Comment:
doesn't seem necessary to store the insights since the consumers are responsible for reacting to any of these events and they don't rely on the history of insights.

Reply:
Done changed that ✅